### ISSUE TITLE
Hyvä Compatibility

### ISSUE THREAD

**Issue by pykettk** [Worldpay/Worldpay-Magento2-CG#117]
# Description
With the [popularity of the Hyva theme](https://trends.builtwith.com/shop/Hyva-Themes) and the [Hyva checkout now in early access](https://www.hyva.io/hyva-checkout.html), are there any plans to make Worldpay compatible?

**Comment by elavarasann** [comment]
Unfortunately no, our module developed with Magento's default Luma theme.

**Comment by pykettk** [comment]
@elavarasann Yes, I know it's not compatible _now_ but I was asking whether there were any plans to _make_ it compatible in the future? I know of a few merchants that have moved away from Worldpay to another payment gateway because of the lack of Hyva support.

**Comment by vladpredaworldpay** [comment]
Hello. We will not be doing in 2025, we're considering it for 2026.
